### ISSUE TITLE
Create CRON job for Coldstunning on core

### ISSUE THREAD

**Issue by matdenkas** [conrad-blucher-institute/semaphore#160]
Runs docker execute against core to run a model
https://docs.docker.com/engine/reference/commandline/exec/

**Comment by lovelysandlonelys** [comment]
Examples I am referencing: 
https://www.airplane.dev/blog/docker-cron-jobs-how-to-run-cron-inside-containers
https://betterstack.com/community/questions/how-to-run-cron-jobs-inside-docker-container/

**Comment by lovelysandlonelys** [comment]
Notes before winter break: 

What I've Done: 
- Created a cron file with the correct timing we chose for testing
- Created a dockerfile for the cron file that copies it to the container and runs it. 

Roadblock: 
- Due to issues with my machine I can't properly test anything at the moment and the initial method that I looked at with cron having its own container maybe isn't suited to the structure we already have set up. 

Thoughts: 
- I think I came at this from the wrong direction? I focused a little bit too much on what the differences would be if we were running a cron job inside of a container because that's what the unknown is for me but I think last meeting you mentioned not having a cron file in the repository and having a script that creates one? I'm not sure how to approach that. 
- Should the cron file just be coppied into the core container?
